File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Python/Parte1/hexagono.py
from figura import Figura
import math

class Hexagono(Figura):
    def __init__(self, lado):
        super().__init__(lado)

    def calcular_area(self):
        return (3 * math.sqrt(3) * self.dimension1 ** 2) / 2

    def calcular_perimetro(self):
        return 6 * self.dimension1

File: 3BV2/PP/P1/Ejercicios/06-Polimorfismo/Java/Parte1/Hexagono.java
public class Hexagono extends Figura {
    public Hexagono(double lado) {
        super(lado);
    }

    @Override
    public double calcularArea() {
        return (3 * Math.sqrt(3) * Math.pow(dimension1, 2)) / 2;
    }

    @Override
    public double calcularPerimetro() {
        return 6 * dimension1;
    }
}
